content-script/toggle.js

```diff
@@ -0,0 +1,43 @@
+import { getColors } from "./highlight";
+
+export function getExistingColor(element) {
+  return element.style.getPropertyValue("background");
+}
+
+export function turnOffHighlight(keyword) {
+  const elements = document.querySelectorAll(`[data-highlight="${keyword}"]`);
+
+  Array.from(elements).forEach((element) => {
+    element.style.removeProperty("background");
+  });
+}
+
+export function turnOffHighlightAll() {
+  const elements = document.querySelectorAll(`[data-highlight]`);
+
+  Array.from(elements).forEach((element) => {
+    element.style.removeProperty("background");
+  });
+}
```

Comment:
p5: 이 두 함수는 중복되는 부분이 있어서 추후에 하나의 함수로 리팩터링하면 괜찮을 것 같습니다.
하지만 함수명을 통해 함수의 기능을 보다 더 명시하고 싶으시면 지금 그대로 유지하는 것도 좋습니다.
```suggestion
export function turnOffHighlight(keyword) {
  const elements = keyword ? 
  document.querySelectorAll(`[data-highlight="${keyword}"]`) :
  document.querySelectorAll(`[data-highlight]`);
  
  Array.from(elements).forEach((element) => {
    element.style.removeProperty("background");
  });
}
```

Reply:
그렇군요. 그럼 MVP가 모두 완료된 이후 리팩터링을 고려해 보도록 하겠습니다. 또는 이후 구현 단계에서 해당 로직이 3번 이상 반복될 상황이 발생할 경우에도 바로 리팩토링을 해볼께요 :)